wiki/Gameplay/Game_modifier/Muted/en.md

```diff
@@ -0,0 +1,51 @@
+---
+stub: true
+tags:
+  - MU
+---
+
+# Muted (lazer mod)
+
+::: Infobox
+
+<!-- lint ignore heading-increment -->
+
+#### Muted
+
+![No Scope mod icon](/wiki/Gameplay/Game_modifier_(lazer)/img/mods/MU.png)
+
+*Can you still feel the rhythm without music?*
+
+|  |  |
+| :-- | :-- |
+| Acronym | MU |
+| Type | Fun |
+| Game modes | ![][osu!] ![osu!taiko] ![][osu!catch] ![osu!mania] |
+| Score multiplier | 1.00x |
+| Status | Ranked |
+| Incompatible mods | None |
+
+:::
+
+*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*
+
+## Description
+
+The **Muted** mod progressively removes the audio between the first and hundredth combo. Beyond that number, only the metronome can be heard for the rest of the map. If you miss an object, everything will be back to normal.
+
+## Customise
+
+![Customise Muted](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/MU.jpg)
+
+- **Start muted:** The audio will be muted when you start the map. The more combo you get, the better you can hear the track. *The option is disabled by default.*
```

Comment:
Would it make sense to create a table for these settings?

Like so maybe:

| Mod setting | Description | Default |
| :-: | :-- | :-: |
| Test | This is a test. | 0 |

Reply:
paragraphs inside tables are pretty bad UX for some clients (mobile) and we have no way to fix that yet